app.js

```diff
@@ -1,10 +1,17 @@
 const express = require('express');
 const app = express();
 
+const { handlePsqErrors, handleCustomErrors, handleServerErrors } = require('./middlewares/error-handlers.js');
 const { getTopics } = require('./controllers/topic-controllers.js');
+const { getEndpoints } = require('./controllers/endpoint-controllers.js');
 
 app.use(express.json());

```

Comment:
It's best practice to only commit code that is necessary/being used. We don't need to use express.json() until we are dealing with a patch/post request. Something to bear in mind going forwards!